sandbox-templates/rehype-dom-debug/src/index.js

```diff
@@ -0,0 +1,27 @@
+// ⚠️ Important! Please make sure the dependencies are up to date.
```

Comment:
This file should be formatted (see above)

Reply:
updated